tests/custom_steps.js

```diff
@@ -1,8 +1,13 @@
+const systemMessageLocator = '//*[@class="page-alert-list"]/.//div[@aria-label]';
+const systemMessageText = '//div[contains(@aria-label, "Alert ")]/div/div[not(*)]';
+const systemMessageButtonClose = '.page-alert-list button';
+
 module.exports = () => actor({
 
   verifyPopUpMessage(message, timeout = 30) {
```

Comment:
I suggest to rename method to "seePopUpMessageWithText" as I.verify... is not expected per codecept common methods for "I" actor